src/type-layout.md

```diff
@@ -73,6 +73,12 @@ at least equal to the size and alignment of a pointer.
 > title="Dynamically Sized Types">DSTs</abbr> are currently twice the size of
 > the size of `usize` and have the same alignment.
 
+## `str` Layout
+
+Strings are laid out so that the `nth` character of the string is offset from the
+start of the string by `n * the size of a character` bytes. A string has a size of
```

Comment:
Why not write `n * size_of::<char>()` here?

Reply:
Do string slices have a length equal to the number of characters?

Reply:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2015&gist=592619fcf323ee9576fa83ce6ee90b76

Reply:
So UTF-8 encoded characters can be anywhere from 1 to 4 bytes?